tests/system/conftest.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
```

Comment:
Almost all of the content of this file is the same as the integration test conftest. I suggest moving repeated code to a general conftest for all the tests.